cleaning_services/demo/crm_lead.xml

```diff
@@ -0,0 +1,134 @@
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="company_id" ref="base.main_company"/>
+    <field name="description"><![CDATA[<div data-oe-version="1.1"</div>]]></field>
+    <field name="priority">2</field>
+    <field name="stage_id" ref="crm.stage_lead3"/>
+    <field name="tag_ids" eval="[(6, 0, [ref('crm_tag_1'), ref('crm_tag_2'), ref('crm_tag_3')])]"/>
+    <field name="expected_revenue">2000.0</field>
+    <field name="date_deadline">2025-02-28</field>
+    <field name="iap_enrich_done" eval="True"/>
+  </record>
+  <record id="crm_lead_2" model="crm.lead">
+    <field name="name">Event Venue Cleanup</field>
+    <field name="partner_id" ref="res_partner_35"/>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="company_id" ref="base.main_company"/>
+    <field name="stage_id" ref="crm.stage_lead2"/>
+    <field name="tag_ids" eval="[(6, 0, [ref('crm_tag_19'), ref('crm_tag_26'), ref('crm_tag_31')])]"/>
+    <field name="expected_revenue">3143.0</field>
+    <field name="probability">97.62</field>
+    <field name="iap_enrich_done" eval="True"/>
+  </record>
+  <record id="crm_lead_3" model="crm.lead">
+    <field name="name">Post-Construction Cleaning</field>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="company_id" ref="base.main_company"/>
+    <field name="stage_id" ref="crm.stage_lead1"/>
+    <field name="tag_ids" eval="[(6, 0, [ref('crm_tag_3'), ref('crm_tag_31')])]"/>
+    <field name="expected_revenue">1259.0</field>
+    <field name="probability">50.0</field>
+    <field name="iap_enrich_done" eval="True"/>
+  </record>
+  <record id="crm_lead_4" model="crm.lead">
+    <field name="name">School Janitorial Service</field>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="company_id" ref="base.main_company"/>
+    <field name="stage_id" ref="crm.stage_lead1"/>
+    <field name="tag_ids" eval="[(6, 0, [ref('crm_tag_18'), ref('crm_tag_29')])]"/>
+    <field name="expected_revenue">3160.0</field>
+    <field name="probability">50.0</field>
+    <field name="iap_enrich_done" eval="True"/>
+  </record>
+  <record id="crm_lead_5" model="crm.lead">
+    <field name="name">Restaurant Kitchen Cleaning</field>
+    <field name="partner_id" ref="res_partner_27"/>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="company_id" ref="base.main_company"/>
+    <field name="stage_id" ref="crm.stage_lead4"/>
+    <field name="tag_ids" eval="[(6, 0, [ref('crm_tag_19'), ref('crm_tag_25'), ref('crm_tag_30')])]"/>
+    <field name="expected_revenue">3367.0</field>
+    <field name="probability">100.0</field>
+    <field name="date_closed">2025-02-26 12:12:24</field>
```

Comment:
For the closed date, you can either define it in the past (relative value) or remove the definition (the date will the one of the installation). Up to you.